### PR TITLE
{bio}[foss-2020b] Biopython bumped to 1.79 and foss-2021a

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2020b-Python-3.8.6.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'Biopython'
+version = '1.79'
+versionsuffix = '-Python-%(pyver)s'
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/b/Biopython/Biopython-1.79-foss-2020b-Python-3.8.6.eb:5]: Erm, yes, you are right. Teaches me to do things late at night.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/ad65379543de8d261d57fc8c510c4f1a for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by sassy-crick** [comment]
We already have that version, courtesy of Robert Mijakovic <robert.mijakovic@lxp.lu>, so we don't need mine here. Closed.

**Pull request closed without merging**